dev/io.openliberty.microprofile.telemetry.2.1.logging.internal/.project

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>io.openliberty.microprofile.telemetry.2.0.logging.internal</name>
```

Comment:
```suggestion
	<name>io.openliberty.microprofile.telemetry.2.1.logging.internal</name>
```
Is this project even needed or can we use the 2.0 bundle?  Is there anything new in the 2.1 bundle that isn't in the 2.0 one?

Reply:
You didn't answer on this one.  Are there opentelemetry changes that require a new logging internal bundle version?  Ideally we would only update those classes that actually are needed to be updated and pulls the other ones from the previous version to avoid the dual maintenance.

Reply:
The 2.0 version of this bundle uses `io.opentelemetry.semconv.SemanticAttributes` but the classes have moved into 
`io.opentelemetry.semconv.ServerAttributes`, `io.opentelemetry.semconv.UserAgentAttributes` etc... We plan to refactor this later. Work has started: https://github.com/OpenLiberty/open-liberty/compare/integration...benjamin-confino:open-liberty:yasmin-telemetry-2.1-2?expand=1